README.md

```diff
@@ -124,6 +128,6 @@ If you run into trouble, please first refer to our [FAQ](https://ark-analysis.re
 If you would like to help make `ark` better, please take a look at our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html). 
 
 ## Citation
-Please cite our paper if you found our repo useful!
+Please cite our paper if you found our repo useful! 
```

Comment:
Will you update the link please https://www.nature.com/articles/s41587-021-01094-0